File: Android/GameControllerSimulator/app2/src/main/java/cn/maoyanluo/gamecontrollersimulator2/bean/FeedbackReceived.kt
package cn.maoyanluo.gamecontrollersimulator2.bean

import com.google.gson.annotations.SerializedName

data class FeedbackReceived(
    @SerializedName("large_motor")
    val largeMotor: Int,
    @SerializedName("small_motor")
    val smallMotor: Int,
    @SerializedName("led_number")
    val ledNumber: Int
)

File: Windows/GameControllerSimulator/CommonLibrary/Bean/FeedbackReceived.cs
using System.Text.Json.Serialization;

namespace CommonLibrary.Bean
{
    public class FeedbackReceived
    {
        [JsonPropertyName("large_motor")]
        public int LargeMotor { get; set; }

        [JsonPropertyName("small_motor")]
        public int SmallMotor { get; set; }

        [JsonPropertyName("led_number")]
        public int LedNumber { get; set; }
    }
}
